lib/pages/scout/DesktopScout.dart

```diff
@@ -729,17 +716,18 @@ class _DesktopScoutState extends State<DesktopScout> {
                             0,
                             Color.fromRGBO(254, 197, 0, 0.2)),
                         child: TextButton(
-                            onPressed: (){
-                              if (kIsWeb){
-                                 showDialog(
-                                    context: context,
-                                    builder: (BuildContext context) => buyDialog(context, athlete));
-                              }else {
-                                setState(() {
-                                  curAthlete = athlete;
-                                  athletePage = true;
-                                });
-                              }
+                            onPressed: () {
+                              // if (kIsWeb) {
+                              //   showDialog(
+                              //       context: context,
+                              //       builder: (BuildContext context) =>
+                              //           buyDialog(context, athlete));
+                              // } else {
+                              //   setState(() {
+                              //     curAthlete = athlete;
+                              //     athletePage = true;
+                              //   });
+                              // }
```

Comment:
So... looks like we need to just get rid of this code right? Otherwise set a TODO and tag the ticket thats going to make this code relevant again